packages/passwordless-auth-construct/src/sign-up/create_user_handler.ts

```diff
@@ -0,0 +1,90 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { CognitoIdentityProviderClient } from '@aws-sdk/client-cognito-identity-provider';
+import { CognitoUserService } from '../services/cognito_user_service.js';
+import { logger } from '../logger.js';
+
+const cognitoCreateUserService = new CognitoUserService(
+  new CognitoIdentityProviderClient()
+);
+
+/**
+ * The Create User for Passwordless SignUp lambda handler
+ * @param event - The event provided by ApiGateway for the request
+ * @returns the response, including status code and response body
+ */
+export const createUser = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  let params: {
+    userPoolId?: string;
+    email?: string;
+    phoneNumber?: string;
+    username?: string;
+  } = {};
+
+  try {
+    params = event && event.body && JSON.parse(event.body);
+  } catch (err) {
+    logger.debug(err);
+
+    return {
+      statusCode: 400,
+      body: 'Invalid parameters',
```

Comment:
should we include stack trace or some other indicator for which parameter is invalid? This is going to be hard to debug otherwise.

should we also assert content type of incoming request?

Reply:
I will change to `Invalid JSON`, the following lines has the individual missed parameters